test/hotspot/jtreg/compiler/intrinsics/math/TestSignumIntrinsic.java

```diff
@@ -25,7 +25,7 @@
 /*
  * @test
  * @summary Test compiler intrinsics for signum
- * @requires os.arch=="aarch64"
+ * @requires os.arch=="aarch64" | os.arch=="x86" | os.arch=="amd64" | os.arch=="x86_64"
```

Comment:
What about os.arch=="i386" ?

Reply:
@DamonFool, I wasn't sure about `os.arch=="i386"`. I'm still getting my head around os.arch and how it is used in these tests. 

I'll add it to the jtreg.